### ISSUE TITLE
Typo in samtools sort command bowtie2 manual page

### ISSUE THREAD

**Issue by ambi1999** [BenLangmead/bowtie2#82]
Hi,

A very minor type in samtools sort command on the following bowtie2 manual page. Basically -o output has not been added.

Page http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml

Present command: samtools sort eg2.bam eg2.sorted

Correct command: samtools sort eg2.bam -o eg2.sorted

ps: I know this is very minor issue but just thought to mention incase it helps someone.

Cheers,
Ambi.

**Comment by BenLangmead** [comment]
Agree that this should be fixed.  The usage in the manual does work for older versions of samtools.  I think it should work for all versions < 1.0.  But for more recent versions, this command should be:

     samtools sort eg2.bam -o eg2.sorted.bam

(note the `.bam` at the end of the output file, since we're specifying the full filename rather than just the prefix, as we did in samtools versions <1.0)

The `bcftools` command in the manual may also need to be updated.  I got `invalid option -- b` when I tried it just now.  Not sure what changed there.
